PrebidMobile/PrebidMobileCore/src/main/java/org/prebid/mobile/core/TargetingParams.java

```diff
@@ -111,6 +113,47 @@ static void fetchLocationUpdates(Context appContext) {
     //endregion
 
     //region Public APIs
+    static final String consentStringKey = "Prebid_GDPR_consent_strings";
+
+    public static void setGDPRConsentStrings(Context context, String string) {
+        if (!TextUtils.isEmpty(string) && context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            SharedPreferences.Editor editor = pref.edit();
+            editor.putString(consentStringKey, string);
+            editor.apply();
+        }
+    }
+
+    public static String getGDPSConsentStrings(Context context) {
+        if (context != null) {
+            SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
+            if (pref.contains(consentStringKey)) {
+                return pref.getString(consentStringKey, "");
+            }
```

Comment:
We need to check for IAB default `IABConsent_ConsentString` as a fallback in else part.